src/python/pants/backend/python/rules/pex.py

```diff
@@ -84,7 +84,7 @@ class Pex(HermeticPex):
 
 # TODO: This is non-hermetic because the requirements will be resolved on the fly by
 # pex, where it should be hermetically provided in some way.
-@rule
+@rule(name="Create PEX")
```

Comment:
```suggestion
@rule(name="Create a pex")
```

Reply:
I'm not sure if PEX should be capitalized or not. It seems like we do it in some places but not others?